packages/callhierarchy/src/browser/callhierarchy-service.ts

```diff
@@ -33,7 +40,33 @@ export class CallHierarchyServiceProvider {
     @inject(ContributionProvider) @named(CallHierarchyService)
     protected readonly contributions: ContributionProvider<CallHierarchyService>;
 
-    get(languageId: string): CallHierarchyService | undefined {
-        return this.contributions.getContributions().find(service => languageId === service.languageId);
+    private services: CallHierarchyService[] = [];
+
+    @postConstruct()
+    init(): void {
+        this.services = this.services.concat(this.contributions.getContributions());
```

Comment:
hello, just curious: why do we concat as services is always empty no ?

Reply:
It just makes it easier because I don't have to worry about the value being undefined or anything being added before @postConstruct. Performance impact is negligible.

Reply:
I see no differences between `services` and `this.contributions.getContributions()` then ?

Reply:
I don't understand the question.